### DISCUSSION TITLE
Get rid of margins on sides and bottom

### DISCUSSION THREAD

**Discussion by two-jays** [zen-browser/desktop#2094] [Ideas]
I would like to have the option not only to get rid of rounded corners but the margins on sides and bottom entirely!
I think i looks much better without an extra border if you have a non white background on the webview.
![grafik](https://github.com/user-attachments/assets/9a65d7fc-2008-4b5c-b558-ff315e33503e)

Of course if you are in split view or have the sidebar active, margins are visually important and should be shown.

**Comment by bkosowski** [comment]
Yes, a simple toggle "Page margins" would be great for this. Right now, the margins are weird. They don't add anything but only take away space from the page. And this present even in full screen mode.

Example screenshot of a full screen page in the compact mode:
![2025-03-09_20-08](https://github.com/user-attachments/assets/17b577ba-7589-4d9a-b576-0f44334dbd24)

**Comment by MrScottyTay** [comment]
Any update on this?

**Comment by MrScottyTay** [comment]
I found a workaround for now. If you go on about:config and search "zen" you should find a variable for the border margin (I'm on my phone currently and can't remember the exact name of it - I'll update this if I remember when I get back to my desktop).

You can change the size with that margin. I found 2 to be really nice. I already advised the roundness slightly so the tighter margin fits a little better too.

This does change it on all sides of the web content pane though, just so you're aware. But I think it looks great with these changes.

**Reply by blakegearin** [reply]
Seeing a couple that you might be referencing...

- `zen.theme.border-radius` 
- `zen.view.show-bottom-border`

**Reply by lajawi** [reply]
> - `zen.theme.border-radius`

From the name, it suggests it sets *some* radius, but which exactly?

I'm specifically looking to change the radius of the website containers.

**Reply by MrScottyTay** [reply]
I don't use zen anymore but iirc the radius is the radius of the rounded corners for the web content

**Reply by lajawi** [reply]
> the radius of the rounded corners for the web content

Exactly my thought, but it doesn't seem to work or change anything. The web content's corners are still as round as before changing the value, even if I use ridiculously high values or even `0`.

**Comment by XanderLeaDaren** [comment]
Set `zen.theme.content-element-separation` to `0` in about:config

**Reply by blakegearin** [reply]
Yes, this appears to be the answer

cc @two-jays

**Reply by bkosowski** [reply]
This is a great find. Would be cool to have it in the normal settings. Would be easier for everyone.

**Reply by savemejeff** [reply]
Worked! Thank you so much

**Reply by kickthedragon** [reply]
Thank you this worked!

**Reply by blakegearin** [reply]
@two-jays Could you mark this as answered?

**Reply by rmgleon** [reply]
thanks!
